src/cmd-buildextend-installer

```diff
@@ -128,24 +123,44 @@ initrd_ignition_padding = 256 * 1024
 
 
 # https://www.kernel.org/doc/html/latest/admin-guide/initrd.html#compressed-cpio-images
-def mkinitrd_pipe(tmproot, destf):
-    findproc = subprocess.Popen(['find', '.', '-mindepth', '1', '-print0'],
-                                cwd=tmproot, stdout=subprocess.PIPE)
+def mkinitrd_pipe(tmproot, destf, compress=True):
+    # The kernel requires that uncompressed appended archives start on a
+    # 4-byte boundary.
```

Comment:
Is there a reference for this?  Seems like would make for interesting reading  :)

Reply:
I don't think there is anything that documents this. The [code](https://github.com/torvalds/linux/blob/47ec5303/init/initramfs.c#L463) is where he found this I think. That being said it would be nice to have a comment that explains a little bit more why this is needed and what happens if we don't have it.

Reply:
[There is!](https://github.com/torvalds/linux/blob/47ec5303/init/initramfs.c#L463)

We had to find it the hard way.  :slightly_frowning_face:

Reply:
Yup, expanded the comment.